.travis.yml

```diff
@@ -0,0 +1,18 @@
+os:
+  - linux
+  - osx
+language: generic
+sudo: required
+dist: trusty
+osx_image: xcode7.3
+install:
+  - echo "DEVELOPMENT-SNAPSHOT-2016-02-25-a" > .swift-version
+  - curl -sL https://gist.github.com/kylef/5c0475ff02b7c7671d2a/raw/621ef9b29bbb852fdfd2e10ed147b321d792c1e4/swiftenv-install.sh | bash
+script:
+  - . ~/.swiftenv/init
```

Comment:
This step can be removed, providing you update the installation route to what's described on https://swiftenv.fuller.li/en/latest/integrations/travis-ci.html